poetry/utils/helpers.py

```diff
@@ -77,3 +81,10 @@ def parse_requires(requires):  # type: (str) -> Union[list, None]
 
     if requires_dist:
         return requires_dist
+
+
+def load_dotenv_if_exists():
+    workdir_path = Path(os.getcwd())
```

Comment:
I think it's worth adding a `settings.commands.dotenv` (or better named) config setting here- as you suggested it would let people set it to an alternative filename (i.e. `poetry.env`). setting it to empty / false could turn it off globally?